### PR TITLE
Made address fields editable

### LINKED ISSUE #125: URGENT fix needed - unable edit location info
**SEVERITY : HIGHEST SEVERITY AND PRIORITY**

**Issue:** Unable to manually edit the location information. I was trying to enter information of someone's parents and not able to edit.

Environment: Firefox on mac os  and Chrome : both tested on macos.

image

image

Steps to reproduce:

**scenario 1:**

1. Allow browser to Access GPU location
2. The location details are auto filled

3. Unable to edit to the location

This is the case in Chrome and FF

**Scenario2:** 

1. Don't allow browser to access location
2. The location details are hidden

image

PS: Power went out and I will not be to confirm.

### PR DESCRIPTION
**Pull request by littlestar642**
Fixes #125

### CODE REVIEW

**Comment by vrthra** [comment]
@littlestar642 Could you please provide the format for entering the GPS co-ordinates on the label? Also You do not validate the GPS entered, could you please validate it before accepting it?

**Comment by vrthra** [comment]
@littlestar642 Do you require all the changes to  floodrelief/settings.py ?

File: floodrelief/settings.py [hunk 1]
```diff
@@ -184,4 +160,4 @@ def get_list(text):
 	os.path.join(BASE_DIR, 'static'),
 )
 
-ADMIN_SITE_HEADER = "Keralarescue Dashboard"
+ADMIN_SITE_HEADER = "test"
```
Comment (vrthra) [inline floodrelief/settings.py]: Do you need this change?

**Comment by salmanulfarzy** [comment]
Changes to ` floodrelief/settings.py` and `.vscode/settings.json` are not related to this PR, I think they should be removed. `settings.py` variables can be handled with `env` file.

File: floodrelief/settings.py (hunk 1 above)
Comment (littlestar642) [reply floodrelief/settings.py]: I guess this was added when i copied the settings from this repo
https://github.com/vigneshhari/keralarescue_test_settings
Is this not relevant here?

**Comment by vrthra** [comment]
#146 is a duplicate of this PR

**Comment by vrthra** [comment]
@littlestar642 When the user now clicks *Requesting for Others*, the entire GPS CO-ordinates is hidden.

**Comment by biswaz** [comment]
Cant merge until the this is fixed
"@littlestar642 When the user now clicks Requesting for Others, the entire GPS CO-ordinates is hidden."

**Comment by vrthra** [comment]
@littlestar642 When you made the field editable, as I mentioned before, there is no validation. The user is at a loss to understand what values to give in that field, especially since the auto-filled value does not look like GPS co-ordinates. Could you at least change the label such that the user knows what format is expected on clicking -- requesting for others? (just give a sample -- e.g 49.25968926590981,7.051553546748877)?

Without validation, the user can enter pretty much any value, and the value may not even be useful for the volunteer.

**Comment by littlestar642** [comment]
Made the changes as requested. Also added a placeholder for coordinates values. @biswaz Please review.

**Comment by vrthra** [comment]
@littlestar642 give me a few minutes; I will update soon.

**Comment by littlestar642** [comment]
@vrthra Can you please tell me the Django username and password?

**Comment by littlestar642** [comment]
@biswaz Can you please help me on this. I am not able to access django admin.

**Comment by vrthra** [comment]
Closing this since #289 provides similar functionallity.

**Pull request closed without merging**